isis/src/qisis/objs/Directory/Directory.cpp

```diff
@@ -902,9 +907,6 @@ namespace Isis {
 
     connect( result, SIGNAL( destroyed(QObject *) ),
              this, SLOT( cleanupMatrixViewWidgets(QObject *) ) );
-             
```

Comment:
Why were these removed? (I'm basically just double-checking to make sure that these were) deliberately removed.)

Reply:
Yeah, sorry - this was an unrelated fix. These connections were not needed.

Reply:
No problem! Just double-checking.